flytekit/models/node_execution.py

```diff
@@ -195,12 +195,12 @@ def to_flyte_idl(self):
             else None,
             task_node_metadata=self.task_node_metadata.to_flyte_idl() if self.task_node_metadata is not None else None,
         )
-        obj.started_at.FromDatetime(self.started_at.astimezone(_pytz.UTC).replace(tzinfo=None))
+        obj.started_at.FromDatetime(self.started_at.astimezone(_timezone.utc.UTC).replace(tzinfo=None))
```

Comment:
bad search-and-replace, right? 
```suggestion
        obj.started_at.FromDatetime(self.started_at.astimezone(_timezone.utc).replace(tzinfo=None))
```